packages/grafana-data/src/tsdoc-metadata.json

```diff
@@ -0,0 +1,11 @@
+// This file is read by tools that parse documentation comments conforming to the TSDoc standard.
+// It should be published with your NPM package.  It should not be tracked by Git.

```

Comment:
@mckn what do you mean? Do we need to push this file to npm when publishing package??

Reply:
oh, that one should be ignored. It is auto-generated by api-extractor. Awesome, findings!